File: Iniciante/1012/1012.c
#include <stdio.h>
 
int main() {
    double A, B, C, pi;
    double triangulo, circulo, trapezio, quadrado, retangulo;

    pi = 3.14159;

    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);
    
    triangulo = (A * C) / 2;
    circulo = pi * (C * C);
    trapezio = ((A + B) * C) / 2;
    quadrado =  B * B;
    retangulo = A * B;

    printf("TRIANGULO: %.3lf\n", triangulo);
    printf("CIRCULO: %.3lf\n", circulo);
    printf("TRAPEZIO: %.3lf\n", trapezio);
    printf("QUADRADO: %.3lf\n", quadrado);
    printf("RETANGULO: %.3lf\n", retangulo);

    return 0;
}

File: Iniciante/Pagina 01/1012/1012.py
a, b, c = map(float, input().split())
pi = 3.14159

triangulo = (a * c) / 2
circulo = pi * (c * c)
trapezio = ((a + b) * c) / 2
quadrado =  b * b
retangulo = a * b

print("TRIANGULO: %.3lf" %triangulo)
print("CIRCULO: %.3lf" %circulo)
print("TRAPEZIO: %.3lf" %trapezio)
print("QUADRADO: %.3lf" %quadrado)
print("RETANGULO: %.3lf" %retangulo)
